### ISSUE TITLE
Logs not being sent back to local in 3.14.0

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#845]
In GitLab by @mcastril on Jul 6, 2022, 11:16

@emchamarro reported that logs are not being transferred back anymore in 3.14.0

`/gpfs/archive/bsc32/esarchive/autosubmit/a28v/tmp`

Could it be linked to #844 ?

**Comment by kinow** [comment]
In GitLab by @mcastril on Jul 6, 2022, 11:24

@eferre1 @portegam

**Comment by kinow** [comment]
In GitLab by @molidg on Jul 8, 2022, 11:29

Hi @dbeltrankyl @mcastril in our case we don't have the logs for the first retrial of the job and the logs are not in local and HPC either. The logs for the last execution of the job are available. Our expid is a4dd and the job that usually fails is DA. The jobid that failed and we don't have logs is 23810033 and we are using AS 3.14

thanks!

FYI: @f-macchia

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jul 18, 2022, 14:18

This happens when the second retrial job starts before autosubmit renames the .cmd.out for the timestamp. 

I've implemented a fix on master that will included in 3.14.0b and 4.0.0b late today or tomorrow.

**Comment by kinow** [comment]
In GitLab by @eferre1 on Jul 28, 2022, 10:44

Hi @dbeltrankyl, I'm not sure if it is related or not, but we are having issues with CCA experiments. In my case I'm using AS/3.14.0b.

I have an experiment (a3s1) with a job (SYNCHRONIZE) that AS marks as running and has already exceeded the wallclock time (30 min), but it is still "running". Connecting to CCA, the logs show that the job finished ~10 min after starting, and the `a3s1_SYNCHRONIZE_COMPLETED` is there, so it seems an issue of not being able to check properly the status of the job. 

I was running this test because another user was having problems with experiments in CCA, so it is not an isolated case, and as I said, I'm not sure if it is related with the logs retrieval issue or if I should open another issue.

If you tell me how the status is checked for CCA experiments I could try to manually check it to see if there is any issue that I'm missing (initially we thought it was due to some ECFS issues that were happening yesterday, but today those are solved and our problem still persists).

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jul 28, 2022, 11:59

Hello, 

I think that the issue #812 will come handy to avoid this issue of disconnection. ( the new wallclock checker.) It is still not in the module tho. 

About the check is done by:
         
`ecaccess-job-list {job_id}`

**Comment by kinow** [comment]
In GitLab by @eferre1 on Jul 28, 2022, 12:21

Ok, thanks, then it might be an issue with CCA because this is the return of the `ecaccess-job-list {job_id}` command:

```
eferre1@ bscesautosubmit02:~> ecaccess-job-list 20868962
     Job-Id: 20868962
   Job Name: a3s1_SYNCHRONIZE.cmd
      Queue: cca
       Host: cca.ecmwf.int
   Schedule: Jul 28 07:54
 Expiration: Aug 04 07:54
  Try Count: 1
     Status: EXEC
    Comment: 
```

But if I login to CCA and try to search the jobs running under my user:

```
c3ef@cca-login4:/scratch/ms/spesiccf/c3ef/a3s1> qstat -u c3ef
c3ef@cca-login4:/scratch/ms/spesiccf/c3ef/a3s1> qscan -u c3ef
qscan: No jobs found
```

So I'm still not sure if the problem comes from AS, connectivity, or that ecaccess commands are not working properly.

**Comment by kinow** [comment]
In GitLab by @portegam on Jul 29, 2022, 10:08

Thanks @eferre1, thanks @dbeltrankyl for looking into this. Does anyone know if there is some workaround that could be applied to prevent the disconnection problem?

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jul 29, 2022, 10:24

You can try 3.14.0b, yesterday was updated with the mentioned in https://earth.bsc.es/gitlab/es/autosubmit/-/issues/845#note_174747

There is now a mechanism that:

* Assign a start date when the job is detected as running.
* When the job is running for longer than 1.30*job.wallclock, AS will look if there is a completed file and put it to "complete" status, if not it will be put into "failed" status.

**Comment by kinow** [comment]
In GitLab by @mcastril on Jul 29, 2022, 10:58

@eferre1 can you contact ECMWF support and tell them about this problem?

**Comment by kinow** [comment]
In GitLab by @portegam on Jul 29, 2022, 11:44

Thanks @dbeltrankyl for the suggestion. We'll try it out.
@aude, we were wondering if you are having similar issues running on cca.

**Comment by kinow** [comment]
In GitLab by @eferre1 on Jul 29, 2022, 12:24

I've contacted ECMWF support asking about this issue, lets see if they know anything more about it.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Sep 30, 2022, 09:41

I think there was pending action from ecmwf, @eferre1 did they fix the issue? 

In any case, were there further issues since the last fix on 3.14.0b?

**Comment by kinow** [comment]
In GitLab by @eferre1 on Sep 30, 2022, 09:59

Yes, about the CCA issue they already fixed it and the eccaccess commands are now working properly.
